frontend/src/Editor/QueryManager/QueryManager.jsx

```diff
@@ -42,20 +41,11 @@ class QueryManagerComponent extends React.Component {
       restArrayValuesChanged: false,
       nextProps: null,
       buttonText: '',
+      renameQuery: false,
     };
 
+    this.newNameForQuery = React.createRef();
```

Comment:
If possible, it can be replaced with `onBlur` event and value can be fetched from the event